playbooks/ansible/roles/samba.setup/tasks/cephfs/main.yml

```diff
@@ -7,7 +7,8 @@
       vars:
         prefix: "{{ role_path }}/tasks/cephfs/"
       with_first_found:
-        - files: "{{ [prefix] | product(config.os[config.nodes[inventory_hostname].os].includes) | map('join') | list }}"
+        - files: >-
+            "{{ [prefix] | product(config.os[config.nodes[inventory_hostname].os].includes) | map('join') | list }}"
```

Comment:
The quotes are not needed. The `>-` at the previous line already indicates this is a text.

Reply:
Fixed

Reply:
Did we merge the right version? I can still see double quotes(`"`). Was that intentionally kept?

Reply:
Ah..nvm. `>-` was removed at the same time.